lib/msal-react-native-android-poc/README.md

```diff
@@ -0,0 +1,98 @@
+# MSAL React Native Proof of Concept (Android)
+**Note: With this library being a proof of concept, it is not recommended to be used for React Native applications intended for production.**
+
+MSAL for React Native is a proof of concept that allows React Native applications to authenticate users with Microsoft work and school accounts (AAD) as well as acquire access tokens from the Microsoft identity platform endpoint in order to access secured web APIs. Currently, this library is only available for the Android platform.
+## Getting started
+
+### Installation
+
+`$ npm install @azuread/msal-react-native-android-poc`
+
+### Register your application in the Azure Portal
+Under Authentication -> Platform configurations, click "Add a platform" and select Android. 
+To generate the base64 signature of your application:
+* MacOS: `keytool -exportcert -alias androiddebugkey -keystore ~/.android/debug.keystore | openssl sha1 -binary | openssl base64`
```

Comment:
We don't support linux?

Reply:
Didn't see an option for Linux in the MSAL faq or Azure portal, but I wonder if it's the same as Windows: https://stackoverflow.com/questions/33073463/generating-release-key-hash-in-linux-osubuntu-android-facebook-sdk

Reply:
Sounds good